apps/files/src/components/LocationPicker/LocationPicker.vue

```diff
@@ -0,0 +1,369 @@
+      if (this.currentAction === 'move') {
+        return this.$gettext('Move here')
+      }
+
+      return null
+    }
+  },
+
+  watch: {
+    $route: {
+      handler: 'navigateToTarget',
+      immediate: true
+    }
+  },
+
+  created() {
+    this.originalLocation = this.target
+
+    if (this.currentAction === 'move') {
+      this.SET_MAIN_CONTENT_COMPONENT(MoveSidebarMainContent)
+    }
+  },
+
+  beforeDestroy() {
+    this.SET_MAIN_CONTENT_COMPONENT(null)
+  },
+
+  methods: {
+    ...mapMutations(['SET_MAIN_CONTENT_COMPONENT']),
+    ...mapActions('Files', ['loadFolder']),
+    ...mapActions(['showMessage']),
+
+    navigateToTarget(target) {
+      if (typeof target === 'object') {
+        target = this.target
+      }
+
+      this.loadFolder({
+        client: this.$client,
+        absolutePath: target || '/',
+        $gettext: this.$gettext,
+        routeName: this.$route.name,
+        loadSharesTree: !this.isPublicPage,
+        isPublicPage: this.isPublicPage
+      })
+    },
+
+    createPath(target) {
+      return (
+        this.basePath +
+        `?action=${this.currentAction}&target=` +
+        encodeURIComponent(target) +
+        this.resourcesQuery
+      )
+    },
+
+    selectFolder(folder) {
+      if (this.isRowDisabled(folder)) {
+        return
+      }
+
+      this.$router.push({ path: this.createPath(folder.path) })
+    },
+
+    leaveLocationPicker(target) {
+      if (this.isPublicPage) {
+        this.$router.push({ name: 'public-link', params: { token: target } })
+
+        return
+      }
+
+      this.$router.push({ name: 'files-list', params: { item: target } })
+    },
+
+    async moveResources() {
+      const errors = []
+      const promise = this.isPublicPage ? this.$client.publicFiles : this.$client.files
+
+      for (const resource of this.resources) {
+        let target = this.target || '/'
+        const resourceName = pathUtil.basename(resource)
+        const exists = this.activeFiles.some(item => {
+          return pathUtil.basename(item.name) === resourceName
+        })
+
+        if (exists) {
+          const message = this.$gettext('Resource with name %{name} already exists')
+
+          errors.push({ message: this.$gettextInterpolate(message, { name: resourceName }, true) })
+
+          continue
+        }
+
+        await promise
+          .move(resource, (target += '/' + resourceName), this.publicLinkPassword)
+          .catch(error => errors.push(error))
+      }
+
+      if (errors.length === 1) {
+        this.showMessage({
+          title: this.$gettext('Move has failed'),
+          desc: errors[0].message,
+          status: 'danger'
+        })
+
+        return
+      }
+
+      if (errors.length > 1) {
+        this.showMessage({
+          title: this.$gettext('Move has failed'),
+          desc: this.$gettext('Multiple resources could not be moved'),
+          status: 'danger'
+        })
+        console.error('Move failed:', errors)
```

Comment:
Debugging leftover?

Reply:
It's left on purpose. Until we have added the logic to notifications to "stack" and then display all of the resources that failed, this is a "workaround" to be able to still see all the errors separately.

Reply:
👍